### ISSUE TITLE
Look into adding support for `OffsetDateTime` field type

### ISSUE THREAD

**Issue by aileftech** [aileftech/spring-boot-database-admin#7]

**Comment by aileftech** [comment]
Hey @agxs, I've been experimenting a bit with OffsetDateTime but I have a few doubts. From what I'm seeing, a OffsetDateTime field is still stored as a datetime without any offset at the database level. I've tested this in MySQL but from what I've read is the same in PostgreSQL.

So what I wanted to ask you is if this understanding is correct from your use case, and if you are using something else to store the offset as well or you are just using OffsetDateTime but the offset is always zero.

**Comment by agxs** [comment]
Yes as I understand it with Postgresql the `timestamp with timezone` and `timestamp without timezone` types (which both map to `OffsetDateTime` and `LocalDateTime` in JDBC respectively) are stored identically in the database, just the times are interpreted differently when they're stored so that the ones with timezones are converted to UTC.

If you need to store the actual timezone you're supposed to add that as a separate column yourself, eg a `text` column that might store `Europe/London` or `+02:00` which you'd apply to the timestamp when you read it.

In my case I'm effectively using it as an `Instant` (which isn't directly supported by the Postgres JDBC driver) and applying any timezone adjustments myself. I need to use `OffsetDateTime` as my entity type as `LocalDateTime` doesn't support adding a timezone to it.

**Comment by agxs** [comment]
Can't wait to try it properly! The admin UI is definitely a feature of Django I miss when working on Spring Boot apps.

**Comment by aileftech** [comment]
Fixed in `main`! Will deploy on Maven in the next few days to add some more stuff before release.

**Comment by aileftech** [comment]
Released `0.1.5` on Maven, so I believe you can finally give it a try in the next few days. I hope it runs fine this time; any other issue, please let me know!

**Comment by agxs** [comment]
I forgot to come back and mention that the changes are all working for me now!
